### ISSUE TITLE
Gems not issued when Plus rank is activated

### ISSUE THREAD

**Issue by joshwenke** [CobaltMC/cobalt-issues#7]
Will retroactively apply 100 gems to all current members who have their Discord linked.

**Comment by AwesomelyEpic1** [comment]
Im not sure if other people have had the same issue or not and I cant test it rn to see if it still is an issue but I have plus activated on MC but I do not have the rank on the discord. I have tried to unlink and relink the two accounts together but it says that are already linked or are not linked depending on whick command I use for /link it says they are already linked and for /unlink it says they were never linked. I know it doesn't change game play or perms being that i am a premium, I am just confused and don't want it to happen to other players too. If you want me to test it again ill do it when I am through with finals. (I posted it here cause it is related to this bug so I didn't want to make a whole new report.)
